haskell/test/Test/Ligo/BaseDAO/Proposal.hs

```diff
@@ -529,13 +487,11 @@ flushNotAffectOngoingProposals
   :: (MonadNettest caps base m, HasCallStack)
   => (ConfigDesc Config -> OriginateFn m) -> m ()
 flushNotAffectOngoingProposals originateFn = do
-  ((owner1, _), _, dao, _, admin) <- originateFn testConfig
-
-
   -- Note: Cannot set to few seconds, since in real network, each
   -- calls takes some times to run. 20 seconds seem to be the ideal.
```

Comment:
Likewise
```suggestion
```
BTW we kinda ignored this here (with the new at-origination value), but let's get to that when we'll need to.